tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = cov-clean,py{310,37}-nodeps,py{310,39,38,37},cov-report
+envlist = py{37,38,39,310}
```

Comment:
The `-nodeps` builds guard against introduction of hard dependencies on third party libraries.

That makes it safer and easier to maintain a library which _allows_ integration with `redis`, `boto3`, and potentially other libraries, but doesn't require them to be present in order to be importable.

Removing this testing may impact the safety of relying on `funcx-common` in client-side software like `funcx` and `funcx-endpoint`, so I would recommend very strongly against doing this.